packages/common/client-utils/package.json

```diff
@@ -11,13 +11,26 @@
 	"license": "MIT",
 	"author": "Microsoft and contributors",
 	"sideEffects": false,
-	"main": "dist/index.cjs",
-	"module": "lib/index.mjs",
+	"exports": {
+		".": {
+			"import": {
+				"default": "./lib/indexNode.mjs",
+				"types": "./lib/indexNode.d.ts"
```

Comment:
```suggestion
				"types": "./lib/indexNode.d.mts"
```

Reply:
Thanks, figured it out more or less at the same time I saw your comment :)